app/views/_includes/footer.njk

```diff
@@ -1 +1,40 @@
-{{ govukFooter({}) }}
+{% set helpHtml %}
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <div class="app-footer__email">
+      <h3 class="govuk-heading-s govuk-!-margin-bottom-1">
+        Email
+      </h3>
+      <p class="govuk-!-font-size-16 govuk-!-margin-bottom-1">
+        <a class="govuk-footer__link" href="mailto:ittmentor.funding@education.gov.uk">
+          becomingateacher@education.gov.uk
```

Comment:
```suggestion
        <a class="govuk-footer__link" href="mailto:becomingateacher@digital.education.gov.uk">
          becomingateacher@digital.education.gov.uk
```

Need to doubt check on `digital.education.gov.uk` through as I am unsure is `becomingateacher@education.gov.uk` setup

Reply:
Good spot and now fixed.